### DISCUSSION TITLE
Session Replay for Mobile - Waitlist

### DISCUSSION THREAD

**Discussion by bruno-garcia** [getsentry/sentry#63138] [Product Spitballin']
# Session Replay is now in Open Beta:

* https://github.com/getsentry/sentry/discussions/74322

------------------------------------------------------------------------------------------------------------

TL;DR: We'll have a preview of Mobile Replay coming out soon.  **fill out this [short form](https://sentry.io/lp/mobile-replay-beta/)** to be part of the first cohort to get access to it.

Sentry launched **Session Replay** almost 1 year ago. The support includes **[any web-based app](https://docs.sentry.io/product/session-replay/getting-started/#supported-sdks),** such as JavaScript, React, Electron (desktop) and Capacitor (mobile), as well as server-rendered ones like Django, Express, Laravel, ASP.NET, Spring and Ruby on Rails.

What about Mobile apps? We’ve [heard your requests](https://github.com/getsentry/sentry-react-native/issues/2703) with all **350 👍** and we’re excited to announce we have officially started development for Session Replay for mobile. 🎉

The plan as of now is to start with native [Android](https://github.com/getsentry/sentry-java/issues/2476) and [iOS](https://github.com/getsentry/sentry-cocoa/issues/2616), including Jetpack Compose and SwiftUI. And also [React Native](https://github.com/getsentry/sentry-react-native/issues/2703). 

Does that align with your expectations? [Flutter came up a lot](https://github.com/getsentry/sentry-dart/issues/1193), is that something you’d like to have? Any other framework/platform? [.NET MAUI](https://github.com/getsentry/sentry-dotnet/issues/2136), Cordova, Xamarin?

Another thing we're planning on is having aggressive PII stripping by default - as it happens on our Web SDK where no text or image leaves the user's browser. This is of course configurable. So you can opt-in to text that's safe to capture. 

On mobile we can safely include images that are part of the app bundle. Since these are just assets to help with app layout and don't include any PII. We hope this aligns with your expectations, please let us know.

Sentry is a developer tool, and as such we want to give you the context you need to fix the hardest problems. What would you like to see with regards to debugging context available with your replays? If there are specific aspects of the product experience or feature requests you want us to nail, please let us know below. 👇

If you want to be one of the first to try out the mobile Replay product once our early preview is available, **fill out this [short form](https://sentry.io/lp/mobile-replay-beta/)**.  (Update 03-04-2024: _We replaced the Google form with a waitlist webpage on sentry.io. All submissions from the original form are recorded._)

**Comment by sidneip** [comment]
Very well!

I submit my interest to participate early preview.

I am awaiting response about 

thanks

**Reply by bruno-garcia** [reply]
Thank you! We're eager to get an alpha ready for testing. We'll keep you posted.

**Comment by ansh** [comment]
Will this be compatible with Expo? Since Expo SDK 50 will abandon the `sentry-expo` package and just directly use `@sentry/react-native`

**Reply by krystofwoldrich** [reply]
Yes, the RN Session Replay will be compatible with Expo except for Expo Go development environment.

**Comment by gauravgautamapptile** [comment]
I want in on the early access.

**Reply by jas-kas** [reply]
Hey @gauravgautamapptile, make sure to submit your info here on [this form](https://forms.gle/tgPHNDUi58KwtuuUA) so we can opt-in your org & contact you once it's available. :)

**Comment by gauravgautamapptile** [comment]
done. thankyou.

**Comment by kud** [comment]
I have submitted the form as well. Thanks.

**Comment by swoofer** [comment]
I've submitted too. Thanks!

**Comment by Matthew-Jimenez** [comment]
Lets go!

**Comment by benjamin-Keller** [comment]
Super excited for replays for .NET MAUI!

**Comment by kud** [comment]
Any news by any chance?

**Reply by bruno-garcia** [reply]
We're working towards this first goal: https://github.com/getsentry/sentry/issues/63255 some PRs point to it

**Reply by kud** [reply]
Thanks!

**Comment by mrtawil** [comment]
+1

**Comment by paulschreiber** [comment]
@bruno-garcia https://sentry.io/lp/mobile-replay-beta/ is throwing 502 errors.

**Reply by kud** [reply]
The page is fancy tho.

**Reply by jas-kas** [reply]
@paulschreiber Ah, a few of our marketing site pages were down last Friday. 😅 They were back up within an hour -- you can add yourself to our waitlist now. :)

**Comment by kud** [comment]
I've already submitted my details but it seems I had to do it again with the new marketing site page.

**Reply by jas-kas** [reply]
Thank you! If you submitted to https://sentry.io/lp/mobile-replay-beta/, we got it :)

**Reply by kud** [reply]
Yep, I've just done it 😌 thanks!

**Comment by ben-squint** [comment]
+1 for Flutter support!

**Comment by eliahilse** [comment]
Flutter support would be awesome

**Comment by kevalkikani180** [comment]
I have submitted the form as well. Thanks.

**Comment by NickPuljic** [comment]
Cant wait for React Native session replay!

**Comment by hammoudma** [comment]
This will be super helpful, just joined the waitlist hopefully we get access soon

**Comment by gaprl** [comment]
On mobile Replay for Flutter, it'd be incredible if we find a way to record both painted widgets by Impeller and native plugins as well. Currently, packages like [screenshot](https://pub.dev/packages/screenshot) will only capture painted widgets, which can be limiting if you have an app that relies heavily on UI rendered by native plugins, such as [google_maps_flutter](https://pub.dev/packages/google_maps_flutter).

**Reply by Artein** [reply]
So true

**Comment by tday** [comment]
Super exciting! We're starting a new project with Sentry error monitoring. It'd be amazing to get this added so we can have everything in one place 🤞

**Comment by ShubhGar** [comment]
I have already fill the form and want to try alpha version of replay on android. I am using Sentry for crash reporting and others and also using replay on Web. Please help to check alpha version of android replay.

**Reply by bruno-garcia** [reply]
Hey @ShubhGar thanks for filling out the form. We'll be making this available to as an open beta and reaching out to everyone next month.

**Reply by eliahilse** [reply]
Open beta meaning that everybody, also outside the waitlist, can join?
Or should we enlist to make sure to get access next month

**Reply by jas-kas** [reply]
@eliahilse Open beta meaning every organization (even out the waitlist) can join :)

**Reply by ShubhGar** [reply]
Hey @bruno-garcia any way to test the current alpha version. as I am analysing some other replay like tools. So it better for me to adopt sentry as we are already using it.

**Reply by bruno-garcia** [reply]
@eliahilse 
> Open beta meaning that everybody, also outside the waitlist, can join?
Beta is only available to Early Adopter orgs, but we're considering opening to all orgs.

> Or should we enlist to make sure to get access next month
We plan to reach out to everyone on the waitlist with the SDK version, instructions etc. So please sign up.

@ShubhGar 
> Hey @bruno-garcia any way to test the current alpha version. as I am analysing some other replay like tools. So it better for me to adopt sentry as we are already using it.

Could you please sign up to the wait-list? We'll reach out soon with instructions. Please add your org slug too

**Reply by tday** [reply]
@bruno-garcia would love to get early access as well since our team is launching a product soon! 🙏 I've signed up on the waitlist already

**Reply by bruno-garcia** [reply]
We'll reach out to everyone on the waitlist soon. But since yesterday it's available to all self serve orgs: https://github.com/getsentry/sentry/discussions/74322

**Comment by hammoudma** [comment]
Is there a timeline for when will this be released? even if just for the beta?

**Reply by jas-kas** [reply]
We plan to open up SR for Mobile to all organizations in open beta in the next 2 weeks. I will be posting an update on this discussion once we do!

**Reply by bruno-garcia** [reply]
It's open to all self serve orgs: https://github.com/getsentry/sentry/discussions/74322

**Comment by jas-kas** [comment]
In case you missed it, Mobile Replay is now available for Android, iOS and React Native in open beta (with the exception of orgs on Enterprise plans): https://github.com/getsentry/sentry/discussions/74322 🎉
